ui/packages/consul-ui/app/components/app-view/index.hbs

```diff
@@ -52,7 +52,10 @@
           <div>
     {{#if authorized}}
               <nav aria-label="Breadcrumb">
-                  <YieldSlot @name="breadcrumbs">{{yield}}</YieldSlot>
+                  <YieldSlot @name="breadcrumbs">
+                    {{document-attrs class="with-breadcrumbs"}}
```

Comment:
Can the name of the slot be used instead of adding a class?

Reply:
Not quite sure what you mean here, but we can go over this later if you like.

You can't target/select slot names with CSS. So the class here is added so we target/select it and alter the positioning of a parent element, it's basically in lieu of some sort of parent selector in CSS. We have to put a class somewhere above the actual breadcrumbs element itself. Does that explain what you mean here?